rocketpool/node/collectors/node-collector.go

```diff
@@ -445,9 +471,31 @@ func (collector *NodeCollector) Collect(channel chan<- prometheus.Metric) {
 	if totalRplAtNextCheckpoint < 0 {
 		totalRplAtNextCheckpoint = 0
 	}
+
+	/*
+	 * Calculates a RPIP-30 RPL reward estimate. Assumes that RPIP-30 has been fully phased in
+	 *
+	 * Formula:
+	 * 		current_node_weight / (current_node_weight + previous_interval_total_node_weight) * estimated_collateral_rewards
+	 *
+	 * This will inevitably be inaccurate due to factors external to the node, such as large nodes running
+	 * several bond reductions, or really anything that can move the needle on total_node_weight. However,
+	 * short of generating a full rewards tree at a regular interval to populate this metric, this is the
+	 * best estimator available.
+	 * RPIP-30 is still being phased in, and this estimator will be more and more accurate as it is, but never fully accurate
```

Comment:
this comment will become out of date in a few weeks, just drop it imo

Reply:
Done